esphome/components/mitsubishi_itp/climate.py

```diff
@@ -67,6 +74,7 @@
 )
 
 CONF_DISABLE_ACTIVE_MODE = "disable_active_mode"
+CONF_ENABLE_KUMO_EMULATION = "kumo_emulation"  # EXPERIMENTAL FEATURE - Enables Kumo packet handling.
```

Comment:
I hate to say this since I know it's probably mentioned in a lot of places, but is there a less Kumo-attached name we can give this?  We're not really trying to emulate the *behavior* of a Kumo, just get the additional data from the MHK (we're emulating the Kumo's responses to do so, so I do get that).  Also I guess it's not called "Kumo" everywhere (though I don't know if these packets will work with MHKs in other regions, I guess).

Reply:
Resolved per your comment - if/when this works, this won't be a toggle and will just exist. Or, at the very least you'll need to opt _out_ of it.